exporters/etw/test/etw_tracer_test.cc

```diff
@@ -116,13 +116,13 @@ TEST(ETWTracer, TracerCheck)
         };
         EXPECT_NO_THROW(innerSpan->AddEvent(eventName3, event3));
         std::this_thread::sleep_for (std::chrono::seconds(1));
-        innerSpan->End();
```

Comment:
Wondering why this is broken?

Reply:
Looks like this comment is on an older / outdated commit?

Reply:
Yes. I was commenting on the line which added `EXPECT_NO_THROW`.

Reply:
Ok got it. That was to keep with consistent with other End() calls,